src/v/pandaproxy/schema_registry/store.h

```diff
@@ -241,6 +241,31 @@ class store {
         }
     }
 
+    /// \brief Return the seq_marker write history of a subject, but only
+    /// config_keys
+    ///
+    /// \return A vector (possibly empty)
+    result<std::vector<seq_marker>>
+    get_subject_config_written_at(const subject& sub) const {
+        auto sub_it = BOOST_OUTCOME_TRYX(
+          get_subject_iter(sub, include_deleted::yes));
+
+        if (sub_it->second.written_at.empty()) {
+            return not_found(sub);
+        }
```

Comment:
I generally prefer a `vassert` for "this can't happen", but given that existing code paths do this, let's keep it, but please add a comment so that it's not too much of a distraction when reading the code.

Reply:
>keep it

👍 

>add a comment

done at top of stack